client/verta/tests/test_endpoint/test_endpoint.py

```diff
@@ -781,7 +780,6 @@ def test_update_from_version_diff_workspace(
         created_entities.append(endpoint)
 
         endpoint.update(model_version, DirectUpdateStrategy(), wait=True)
-        assert endpoint.workspace != model_version.workspace
```

Comment:
Were these two tests failing here? 🤔 I think these checks are important to keep, because the purpose of the test is that the workspaces are different.

Reply:
reverted these changes. Now it is working. Sorry, I think I misunderstood these tests before and I do not know why it was failing before: maybe there was a different MDB version.